src/Sql/Platform/Mysql/Ddl/CreateTableDecorator.php

```diff
@@ -163,14 +163,13 @@ private function normalizeColumnOption($name)
     }
 
     /**
-     * @internal @private this method is only public for PHP 5.3 compatibility purposes.
      *
      * @param string $columnA
      * @param string $columnB
      *
      * @return int
      */
-    public function compareColumnOptions($columnA, $columnB)
+    private function compareColumnOptions($columnA, $columnB)
```

Comment:
Yep, it was me: https://github.com/zendframework/zend-db/commit/bf6d9fc0ad72137739e7e51a3113bf6e32ce39cb